README.md

```diff
@@ -6,51 +6,43 @@ We run an [internal server](./internal-server.md) so private datasets can be kep
 
 We have have general experience [tracking neuroimaging data with git-annex](./git-annex.md).
 
-The data curation scripts should be placed inside the BIDS datasets, under the `code/` folder. See [BIDS specification](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#code). For more convenience, you can create a PR with a curation script that you are working on, so that others can give feedback; once the script is validated, you can simply close the PR and delete the branch without merging.
+# ⚠️ Every dataset should have the following files 
 
-Every dataset should have a README inside it. Example:
-~~~
-# <NAME OF DATASET>
-
-<PLEASE MODIFY>This is an MRI dataset acquired in the context of the XX project. It also contains manual segmentation of MS lesions from expert raters.
-
-Dataset shared by: <NAME AND EMAIL>
-<IF THERE WAS EMAIL COMM>Email communication: <DATE OF EMAIL AND SUBJECT>
-<REPOSITORY OF PROJECT/MODEL, etc>Repository: https://github.com/ivadomed/model_seg_ms_mp2rage
+```
+├── README.md
+├── participants.json
+├── participants.tsv
+├── dataset_description.json
+└── code
+    └── <script_you_used_for_dataset_curation>.py
```

Comment:
A suggestion: I'm learning that [BIDS allows including the original data files in whatever format](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#source-vs-raw-vs-derived-data) under a `sourcedata/` directory. And since we use `git-annex`, a simple `git clone` wouldn't download these large files unless they're explicitly requested with a `git annex get sourcedata/`. Would this be a good workflow to consider?
```suggestion
└── sourcedata
    └── <DICOM files or whatever>
└── code
    └── <script_you_used_for_dataset_curation>.py
```

Reply:
Alternatively, if we want to use `git submodule` instead of `git annex`, this workflow could also be accomplished by storing the DICOMs in a different git repository on `data.neuro.polymtl.ca`, like `data.neuro.polymtl.ca:raw/my_data_set`, and making the `sourcedata/` folder inside `data.neuro.polymtl.ca:datasets/my_data_set` be a git submodule. This would very nicely track provenance.

Reply:
For some of our datasets, I agree this approach is attractive. For other datasets, there are additional complications (eg: source should not be accessible by all students, anonymization/defacing, one source being used for multiple derived BIDS datasets, etc.).

Reply:
I vote strongly for loose-coupling here!

Storing `sourcedata/` in the same repo immediately runs afoul of anonymization rules. It also bloats downloads since while, yes, you *can* use `git annex get sub-001 sub-002 sub-003 ...`, most people will do `git annex get .`. I am pretty confident it does not have any way of excluding folders like `git annex get !sourcedata/`. Storing source data in a separate repo with a pointer avoids both problems. Also just in principle, sourcedata feels like a different concept than BIDS data.

I am doubtful `git submodule` is a good choice, just because UI-wise it is difficult, but having a process that record a URL to upstream datasets (or code) is important. I have had [this issue](https://github.com/neuropoly/data-management/issues/136) open for a while to discuss what actual technology we can standardize on with the least friction. And eventually we'll want to write/find a linter that periodically scans all the data we have to detect broken provenance links.

(btw, @mguaypaq, we can combine `git-annex` and `git submodule` -- that's exactly what Datalad recommends. They're not in opposition. Even if they both give ways to reduce downloads)